vw_golf_mk4.dbc

```diff
@@ -303,7 +303,8 @@ BO_ 1408 Motor_Flexia: 8 XXX
  SG_ Multiplex_Schalter_Motor_Flexia M : 0|1@1+ (1,0) [0|0] "" XXX
 
 BO_ 1416 Motor_7: 8 XXX
- SG_ Frei_Motor_7_3 : 40|24@1+ (1,0) [0|0] "" XXX
+ SG_ Oltemperatur : 56|8@1+ (1,0) [0|0] "" XXX
+ SG_ Frei_Motor_7_3 : 40|16@1+ (1,0) [0|0] "" XXX
```

Comment:
Let's revert this change. If we're going to clean up sorting of signals, they would go in ascending order rather than descending, and it should be done in a standalone PR for that purpose rather than intermixed with translation/commentary updates. And I'm not in a great hurry to perform/review that sort at this particular time.

Reply:
@jyoung8607 let's confirm it's not a misunderstanding. I belive this is NOT a sorting change here.
"Frei_Motor_XX_XX" in both cases seems like "unused unmapped areas" and in both cases I am reducing the lenght of "unused unmapped Frei_Motor" while adding a new field definition which takes a part of what was "unmapped"

like here I believe that I am ading Oltemperatur and reduce unmapped placeholder from 24 bits to 16 bits

Question: shall I revent addition of the new field?